### ISSUE TITLE
Csprojes are not indexed properly [source.dot.net]

### ISSUE THREAD

**Issue by hankovich** [KirillOsenkov/SourceBrowser#208]
I've noticed significant differences between csprojes in source.dot.net and in the dotnet repositories. 

Some examples:
https://source.dot.net/#Microsoft.AspNetCore.Authentication.JwtBearer/Microsoft.AspNetCore.Authentication.JwtBearer.csproj
https://source.dot.net/#System.Linq/System.Linq.csproj

All cspject still have `netcoreapp2.1` as TFM.

**Comment by KirillOsenkov** [comment]
I think this is by design. The .csproj files are artificially cleaned up and generated to ensure the indexing process can run in isolation from the actual build. @alexperovich can confirm, he wrote source.dot.net.

**Comment by alexperovich** [comment]
Yes, this is by design. The projects and build system in all the dotnet repos are different enough and don't always have synchronized versions of dependencies that the indexer can't load all of them in the same process. I generate csprojs based on the csc commands found in the binlog to ensure that the compiler gets all the proper inputs for the indexing process.
